lamindb/curators/__init__.py

```diff
@@ -402,8 +402,18 @@ def validate(self) -> None:
                 self._cat_manager_validate()
             except pandera.errors.SchemaError as err:
                 self._is_validated = False
+                err_msg = str(err)
+
+                # Reduce Pandera validation stdout for many columns
+                if "column" in err_msg and "not in dataframe" in err_msg:
+                    missing_col = err_msg.split("column '")[1].split("'")[0]
+                    display_cols_str = _format_values(
+                        list(self._dataset.columns), n=10, quotes=True, sep="'"
```

Comment:
Could this be simplified to `display_cols_str = _format_values(self._dataset.columns, n=10)`?